File: benchmark/results/java-invocation-script/run-004/b-baseline/output/com/metaengine/demo/identity/services/AuthService.java
package com.metaengine.demo.identity.services;

import com.metaengine.demo.identity.aggregates.User;
import java.util.List;

/** AuthService domain service in the identity domain. */
public class AuthService {
    /** Authenticate and create a User from a partial input. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a User by id, or null if not found. */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Users up to limit. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a User by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-003/b-baseline/output/metaengine_demo/identity/services/auth_service.py
"""AuthService for the identity domain."""
from typing import Optional

from metaengine_demo.identity.aggregates.user import User


class AuthService:
    """AuthService domain service."""

    def create(self, input: Optional[User]) -> User:
        """Authenticate a new user registration."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[User]:
        """Look up an authenticated user by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[User]:
        """List authenticated users."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Revoke authentication for a user."""
        raise NotImplementedError("not implemented")
